src/main/groovy/com/athaydes/gradle/ceylon/CeylonPlugin.groovy

```diff
@@ -25,40 +26,40 @@ import org.gradle.language.base.plugins.LanguageBasePlugin
 @CompileStatic
 class CeylonPlugin implements Plugin<Project> {
 
-    static final Logger log = Logging.getLogger( CeylonPlugin )
+    static final Logger log = Logging.getLogger(CeylonPlugin)
 
     @Override
-    void apply( Project project ) {
+    void apply(Project project) {
```

Comment:
I would appreciate if you did not modify the code style I use.

Get IntelliJ or your IDE to apply a whitespace within parentheses and auto-format again.